src/hook.ts

```diff
@@ -69,13 +69,21 @@ async function sendRequestWithRetries(
     }
 
     if (timeSinceTokenCreationInMs >= FIVE_SECONDS_IN_MS) {
+      if (retries > 0) {
+        error.message = `[@octokit/auth-app] After ${retries} retries within ${
+          timeSinceTokenCreationInMs / 1000
+        }s after creating the installation access token, the response remains 401. At this point it's unlikely a replication lag, but a legit authentication problem or a system outage, please check https://www.githubstatus.com/`;
+      }
```

Comment:
this is the only relevant change, the changes below are just reformatting for readability

Reply:
The comment on [line 52](https://github.com/octokit/auth-app.js/pull/168/files#diff-298fdbe874b8414acb60ef615c102ec8R52) should be updated to 5 seconds instead of 1 minute.

Reply:
Good catch 👍🏼